experimental/uefi/src/main.rs

```diff
@@ -0,0 +1,66 @@
+//
+// Copyright 2022 The Project Oak Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+#![no_main]
+#![no_std]
+#![feature(abi_efiapi)]
+#![feature(custom_test_frameworks)]
+#![test_runner(crate::test_runner)]
+#![reexport_test_harness_main = "test_main"]
```

Comment:
Where does the `test_main` string come from? Is it important that it stays like that, or can it be changed? Please add some details

Reply:
I assume it can be changed; I've added a link to the blog post this code draws upon heavily for the testing infrastructure.